docs/common-problems.asciidoc

```diff
@@ -96,6 +97,48 @@ and it alleviates problems that might result from sudden spikes of data.
 You can increase the queue size by overriding `queue.mem.events`.
 Be aware that increasing `queue.mem.events` can significantly affect APM Server memory usage.
 
+[float]
+[[delete-old-data]]
+==== Delete old data
+
+A late symptom of a full queue is that Elasticsearch ran out of disk space and started rejecting insertions.
```

Comment:
We are talking about the internal queue, right? I'd make this statement a bit more direct and say something like:

A full internal queue might cause Elasticsearch to run out of disk space and start rejecting insertions.

Reply:
Thanks for taking the time to review the rest as well.
So this is the other way around: Elasticsearch runs out of disk space, and because of it it starts rejecting insertions, and because of it the queue fills up (because nothing gets pulled out). 
Users then get this "queue is full" message in their app logs. 
In this case the root cause is that the disk that elasticsearch is writing to is full.
How would you phrase it better?

Reply:
Ah, that makes sense! The original sentence was  a little ambiguous. :-) How about something like:

The internal queue might be full because Elasticsearch ran out of disk space and started rejecting insertions. If this happens, you will likely need to remove old indices to make room for new data.